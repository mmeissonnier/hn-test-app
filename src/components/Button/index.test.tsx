import React from 'react';
import { render } from '@testing-library/react';
import { Button } from '.';

describe('components/Button', () => {
  it('should render correctly', () => {
    const { container } = render(<Button>FOO</Button>);
    expect(container).toMatchSnapshot();
  });
});
