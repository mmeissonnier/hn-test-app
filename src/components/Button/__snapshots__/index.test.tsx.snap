// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Button should render correctly 1`] = `
.c0 {
  background-color: transparent;
  padding: 0;
  border: none;
  white-space: nowrap;
}

.c0:hover {
  cursor: pointer;
  -webkit-text-decoration: underline;
  text-decoration: underline;
  color: #FF742B;
}

<div>
  <button
    class="c0"
  >
    FOO
  </button>
</div>
`;
